src/v2/pages/about/RoadmapPage/components/GoalMeter/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+
+import Box from 'v2/components/UI/Box'
+import Text from 'v2/components/UI/Text'
+import Meter from 'v2/components/UI/Meter'
+
+interface GoalMeterProps {
+  currentMrr: number
+  goalMrr: number
+  goalDate: string
+}
+
+export const GoalMeter: React.FC<GoalMeterProps> = ({
+  currentMrr,
+  goalMrr,
+  goalDate,
+}) => {
+  return (
+    <React.Fragment>
```

Comment:
Been preferring the `<></>` syntax for fragments.

Reply:
++